### PR TITLE
Added wiki commands and plugin commands

### PR DESCRIPTION
**Pull request by zneeley**
I have added new commands for all the machines, items, ores on the wiki. I have also added commands for the spigot page and resource pack git repo.

### CODE REVIEW

File: commands/plugin/resoure_pack_link.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'resourcePack',
+			aliases: ['resourcePack'],
+			group: 'wiki',
```
Comment (brettsaunders21) [inline commands/plugin/resoure_pack_link.js]:
```suggestion
			group: 'plugin',
```
The above command is in the plugin folder, either move to wiki folder or change group as above.

File: commands/wiki/items/configurator.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'configurator',
+			aliases: ['config'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/configurator.js]:
```suggestion
			group: 'wiki-item',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item  or just items

File: commands/wiki/items/cores.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'cores',
+			aliases: ['cores'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/cores.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/drill.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'drill',
+			aliases: ['drill'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/drill.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/engineers_hammer.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'engineersHammer',
+			aliases: ['engHammer'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/engineers_hammer.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/excavator.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'excavator',
+			aliases: ['excavator'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/excavator.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/power_hammer.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'powerHammer',
+			aliases: ['hammer'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/power_hammer.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/sickle.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'sickle',
+			aliases: ['sickle'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/sickle.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/items/wrench.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'wrench',
+			aliases: ['wrench'],
+			group: 'itemWiki',
```
Comment (brettsaunders21) [inline commands/wiki/items/wrench.js]:
```suggestion
			group: 'wiki-items',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-item or just items

File: commands/wiki/machines/block_breaker.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'blockBreaker',
+			aliases: ['breaker'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/block_breaker.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/block_placer.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'blockPlacer',
+			aliases: ['placer'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/block_placer.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/electric_furnace.js [hunk 12]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'electricFurnance',
+			aliases: ['furnance'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/electric_furnace.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/energy_cell.js [hunk 13]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'energyCell',
+			aliases: ['cell'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/energy_cell.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/foundry.js [hunk 14]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'foundry',
+			aliases: ['foundry'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/foundry.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/macerator.js [hunk 15]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'macerator',
+			aliases: ['macerator'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/macerator.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/magnatisation_table.js [hunk 16]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'magnatisationTable',
+			aliases: ['magTable'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/magnatisation_table.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/magnatiser.js [hunk 17]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'magnatiser',
+			aliases: ['magnatiser'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/magnatiser.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/solar_panel.js [hunk 18]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'solarPanel',
+			aliases: ['solar'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/solar_panel.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/machines/solid_fuel_generator.js [hunk 19]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'solidFuelGenerator',
+			aliases: ['generator'],
+			group: 'machineWiki',
```
Comment (brettsaunders21) [inline commands/wiki/machines/solid_fuel_generator.js]:
```suggestion
			group: 'wiki-machines',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-machines or just machines

File: commands/wiki/ores/copper_ore.js [hunk 20]
```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'copperOre',
+			aliases: ['copper'],
+			group: 'oreWiki',
```
Comment (brettsaunders21) [inline commands/wiki/ores/copper_ore.js]:
```suggestion
			group: 'wiki-ores',
```
Group names must be lower case and must be identical to the folder path. Please change this and the folder it's in to be commands/wiki-ores or just ores

**Review by brettsaunders21 — CHANGES_REQUESTED** [review]
Please can you fix the attached issues.

Also, none of these commands will work, you need to add the command groups for plugin, items, ores and machines to the main index.js file. The link bellow will help and there is already two examples of this in index.js
https://discordjs.guide/commando/#creating-your-index-js-file

**Comment by zneeley** [comment]
I properly added the commands and created embeds for all of the commands.

File: commands/wiki-items/configurator.js [hunk 21]
```diff
@@ -0,0 +1,29 @@
+const { Command } = require('discord.js-commando');
+
+const wrenchEmbed = {
```
Comment (brettsaunders21) [inline commands/wiki-items/configurator.js]:
```suggestion
const configuratorEmbed = {
```

File: commands/wiki-items/configurator.js [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+			name: 'configurator',
+			aliases: ['config'],
+			group: 'wiki-items',
+			memberName: 'configurator',
+			description: 'Replies with a link to the Configurator wiki.'
+		});
+	}
+
+	run(message) {
+		return message.say({ embed: wrenchEmbed });
```
Comment (brettsaunders21) [inline commands/wiki-items/configurator.js]:
```suggestion
		return message.say({ embed: configuratorEmbed });
```

**Review by brettsaunders21 — CHANGES_REQUESTED** [review]

**Review by brettsaunders21 — APPROVED** [review]

**Pull request merged**